jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitProjectBuilderCommon.kt

```diff
@@ -38,10 +42,17 @@ abstract class SamProjectTemplate {
 
     open fun postCreationAction(runtime: Runtime, contentRoot: VirtualFile, rootModel: ModifiableRootModel) {
         SamCommon.excludeSamDirectory(contentRoot, rootModel)
-
+        openReadmeFile(contentRoot, rootModel.project)
         createRunConfigurations(contentRoot, rootModel.project)
     }
 
+    private fun openReadmeFile(contentRoot: VirtualFile, project: Project) {
+        VfsUtil.findRelativeFile(contentRoot, "README.md")?.let {
+            val fileEditorManager = FileEditorManager.getInstance(project)
+            fileEditorManager.openTextEditor(OpenFileDescriptor(project, it), true) ?: LOG.warn("Failed to open README.md")
```

Comment:
Odd, this should be a ktlint failure due to not using {} syntax.

Reply:
Changed.

Reply:
https://github.com/aws/aws-toolkit-jetbrains/pull/998 fixes ktlint